pkg/app/web/src/components/applications-page/add-application-drawer/index.tsx

```diff
@@ -72,6 +78,9 @@ export const AddApplicationDrawer: FC<AddApplicationDrawerProps> = memo(
             {...formik}
             title={`Add a new application to "${projectName}" project`}
             onClose={handleClose}
+            onAddFromGit={() => {
+              setShowConfirmToAddFromGit(true);
+            }}
```

Comment:
```suggestion
            onAddFromGit={() => setShowConfirmToAddFromGit(true)}
```
nits, just as in L109